vars/other.groovy

```diff
@@ -1,3 +1,42 @@
+/**
+* Building an execution map of pipeline stages.
+* By default, all the stages are marked as 'true', executing the whole flow.
+* By providing a name of specific stage, all previous stages will be skipped(marked as 'false').
+* In case we want to execute only single stage, the one will be marked as 'true', and all the rest as 'false'.
+*
+* @param start_stage String specifying the name of specific stage.
+* @param only_stage Boolean indicate whether we want to execute a single stage.
+* @return Map map of pipeline stages. Key=<stage_name>, Value=<Boolean>(true=execute, false=skip)
+*/
+def get_v2v_current_stage(start_stage, only_stage) {
+  def stages = [
+                'Create VMs':true,
+                'Install Nmon':true,
+                'Add extra providers':true,
+                'Set RHV provider concurrent VM migration max':true,
+                'Conversion hosts enable':true,
+                'Configure oVirt conversion hosts':true,
+                'Configure ESX hosts':true,
+                'Create transformation mappings':true,
+                'Create transformation plans':true,
+                'Start performance monitoring':true,
+                'Execute transformation plans':true,
+                'Monitor transformation plans':true,
+                'Stop performance monitoring':true
+                ]
+
+  for (e in stages) {
```

Comment:
could you please change the "e" to something more readable like stage

Reply:
'e' stands for 'element' . It is kind of convention when iterating through maps/dict keys.
But I will accept your comment